app/scripts/directives/resources.js

```diff
@@ -1,6 +1,75 @@
 'use strict';
 
 angular.module('openshiftConsole')
+  .directive('containerStatuses', function($filter) {
+    return {
+      restrict: 'E',
+      scope: {
+        containerStatuses: '=',
```

Comment:
Let's rename this `pod` since it's taking in a pod, not the containerStatuses array